libnavigation-core/src/test/java/com/mapbox/navigation/core/accounts/MapboxNavigationAccountsTest.kt

```diff
@@ -0,0 +1,84 @@
+package com.mapbox.navigation.core.accounts
+
+import android.content.Context
+import android.os.Bundle
+import androidx.test.core.app.ApplicationProvider
+import com.mapbox.android.accounts.v1.AccountsConstants
+import org.junit.After
+import org.junit.Assert.assertEquals
+import org.junit.Assert.assertNotNull
+import org.junit.Before
+import org.junit.Test
+import org.junit.runner.RunWith
+import org.robolectric.RobolectricTestRunner
+import org.robolectric.annotation.Config
+
+@RunWith(RobolectricTestRunner::class)
+@Config(manifest = Config.NONE)
+class MapboxNavigationAccountsTest {
+
+    val ctx = ApplicationProvider.getApplicationContext<Context>()
+
+    @Before
+    fun setUp() {
```

Comment:
I'm not a big fan of inline setup in tests, but that's personal preference, we can discuss further as a team 😉 

Reasons why 👇 

👀 [_Working Effectively with Unit Tests_](https://leanpub.com/wewut/read) 
> when writing tests you should prefer DAMP (Descriptive And Maintainable Procedures) 

That includes a bunch of _procedures_ as [Inline Setup](https://leanpub.com/wewut/read#leanpub-auto-inline-setup) 😅

**1. setUp() method**
- When you remove Setup you may limit the number of times you need to check variables when you first encounter a test. So then you no longer need to worry about irrelevant instances from other tests
- It's a waste of time because you normally need to understand all the Setup each time you have to deal with the test
- Setup advocates argue because if you don't use it you normally create duplication. You can solve it using private methods or builders

Normally is good to structure the tests following the triple A 👀 https://speakerdeck.com/guardiola31337/elegant-unit-testing-mobilization-2016?slide=40 http://wiki.c2.com/?ArrangeActAssert
If the `A`rrange part gets too long we should create factory or builder methods. Problem with Setup is that hides the _Arrange_ part completely.